haiku-apps/chat_o_matic/chat_o_matic-0.0.2.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A multi-protocol IM client"
+DESCRIPTION="Chat-O-Matic is a multi-protocol chat client. It supports \
+IRC, XMPP, and other protocols through libpurple."
+HOMEPAGE="https://github.com/JadedCtrl/Chat-O-Matic"
+COPYRIGHT="2009-2011 Andrea Anzani
+	2010-2015 Dario Casalinuovo
+	2009-2011 Pier Luigi Fiorini
+	2021 Jaidyn Levesque"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/JadedCtrl/Chat-O-Matic/archive/refs/tags/v$portVersion.tar.gz"
+CHECKSUM_SHA256="a6f38627c8a12ce82ad8a3296fe6ab8524a7c516f76165d848ba33573270572c"
+SOURCE_DIR="Chat-O-Matic-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	chat_o_matic = $portVersion
+	app:Chat_O_Matic = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libexpat$secondaryArchSuffix
+	lib:libglib_2.0$secondaryArchSuffix
+	lib:libgloox$secondaryArchSuffix
+	lib:libpurple$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	chat_o_matic${secondaryArchSuffix}_devel = $portVersion
+	"
+REQUIRES_devel="
+	haiku${secondaryArchSuffix}_devel
+	chat_o_matic$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libexpat$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgloox$secondaryArchSuffix
+	devel:libpurple$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
```

Comment:
Missed this one :)

This fixes the find for `glib.h`, created a PR upstream to add the same search for `libpurple's headers`, still leaves the failing build issue reported at your github page


```diff
- 	cmd:pkg_config
+ 	cmd:pkg_config$secondaryArchSuffix
```